in_command.go

```diff
@@ -191,18 +197,13 @@ func (c *InCommand) downloadFile(url, destPath string) error {
 	return nil
 }
 
-func (c *InCommand) resolveTagToCommitSHA(tag string) string {
+func (c *InCommand) resolveTagToCommitSHA(tag string) (string, error) {
 	reference, err := c.github.GetRef(tag)
 
-	if err != nil {
-		fmt.Fprintln(c.writer, "could not resolve tag '%s' to commit: %v", tag, err)
-		return ""
-	}
-
-	if *reference.Object.Type != "commit" {
-		fmt.Fprintln(c.writer, "could not resolve tag '%s' to commit: returned type is not 'commit' - only lightweight tags are supported", tag)
-		return ""
+	if err != nil && *reference.Object.Type != "commit" {
+		fmt.Fprintln(c.writer, "could not resolve tag '%s' to commit: returned type is not 'commit' - only lightweight tags are supported")
+		return "", nil
 	}
 
-	return *reference.Object.SHA
+	return *reference.Object.SHA, err
```

Comment:
There should probably be a check like this:

```
if err != nil {
    return "", err
}
```

Otherwise the dereference on this line will panic. Better to guard early and have an explicit happy path to avoid things like that.